iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -2033,6 +2035,14 @@ public TSStatus dropPipe(String pipeName) {
         : status;
   }
 
+  @Override
+  public TSStatus extendDropPipe(TDropPipeReq req) {
```

Comment:
Can we directly name it "dropPipe" and mark the original one as deprecated?

Reply:
Same as the other methods with "extend" in their names.

Reply:
The naming related to ‘extend’ in ConfigManager has been changed, but two functions with the same name are not allowed in the thrift file, so the RPC function can only be named ExtendDropPipe only.